src/v3/src/img/yubikeyDemo.svg

```diff
@@ -0,0 +1,19 @@
+<svg width="91" height="85" viewBox="0 0 91 85" fill="none" xmlns="http://www.w3.org/2000/svg" aria-labelledby="yubikeyDemo">
```

Comment:
@alexdahl-okta This image isn't considered an "icon" is it? i.e., it shouldn't be theme-able, right?